source/ore/routes/list-users.rst

```diff
@@ -0,0 +1,83 @@
+==========
+List Users
+==========
+
+**GET /api/users**
+
+Returns a list of users based on given criteria.
```

Comment:
We should maybe state in which situations this method is and isn't supposed to be used. 
Crawling user data -> bad.